Orange/widgets/visualize/utils/widget.py

```diff
@@ -403,8 +402,6 @@ def __init__(self):
         self.__pending_selection = self.selection
         self._invalidated = True
         self._domain_invalidated = True
-        self.input_changed.connect(self.set_input_summary)
-        self.output_changed.connect(self.set_output_summary)
```

Comment:
Are the signals (`input_changed` and `output_changed`) still needed?

Reply:
You tell me.

I'd prefer if they're eliminated. (Otherwise every widget could have them.)

Reply:
Please keep them. I know we separated the repositories, but we need the API between canvas/widget-base. I'd love to be able to read/write a widget's title from OWWidget code, too.

Reply:
@irgolic, this signal doesn't help you a lot. This is `OWDataProjectionWidget`, not `OWBaseWidget`. You want to keep it here or have it there?

Reply:
Sorry, I got it confused with this. https://orange-canvas-core.readthedocs.io/en/0.1.16/orangecanvas/scheme.events.html#orangecanvas.scheme.events.WorkflowEvent.OutputLinkAdded
By all means, remove these signals. :)